### PR TITLE
build: update to support Swift 5

### PR DESCRIPTION
**Pull request by ikait**
Changes with this PR make build available with Xcode 10.2 and Swift 5.

### CODE REVIEW

**Comment by sherwinski** [comment]
Hi @ikait thanks for the PR. Just wanting to be sure, how will this affect our users who might be using older versions of XCode and/or Swift?

**Comment by ikait** [comment]
Hi @sherwinski. Swift 5 isn’t compatible (cannot build) with older versions of Xcode and Swift. After merge this, I suppose it would be better to make and publish a new tag like 0.5.0 and add comments like "The imgix Swift client is compatible with Swift 5.0" for users who use older versions just like https://github.com/imgix/imgix-swift/blob/master/README.md#swift

**Comment by sherwinski** [comment]
Hi @ikait, that makes sense. I tried to search around and see how other swift projects handle multiple swift/xcode versions but wasn't able to find a concrete practice.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode9.4
+osx_image: xcode10.2
 before_install:
   - rvm get head
```
Comment (sherwinski) [inline .travis.yml]: Could you please remove this entire `before_install` block? We have had issues in the past with `rvm get head` breaking travis builds. Thanks!

**Review by sherwinski — CHANGES_REQUESTED** [review]

**Comment by ikait** [comment]
Hi @sherwinski, thanks for reply. Removed.

**Comment by fumito-ito** [comment]
@sherwinski any updates for this PR ?

**Comment by sherwinski** [comment]
Hi @fumito-ito thanks for following up. My biggest hesitation for merging this PR right now is whether it will prevent us from being able to release features for our users who are still on swift version 4. Additionally, if I had some way of understanding how many swift developers out there are still using swift 4 versus how quickly they are upgrading to swift 5, it would help a lot in deciding when we should upgrade as well.
I'm sure that you and @ikait are much more familiar with the nuances of swift package development, so any insights you might have would be really appreciated. Thanks

**Comment by ikait** [comment]
Hi @sherwinski , Xcode 11.X has been released and I suppose almost iOS projects have already developed by using Swift 5 so I think it's time to merge this branch. Thanks.

**Comment by sherwinski** [comment]
Hi @ikait! I think you are correct and after looking around at a few popular libraries online (e.g. [Alamofire](https://github.com/Alamofire/Alamofire), [Perfect](https://github.com/PerfectlySoft/Perfect), [SwiftLint](https://github.com/realm/SwiftLint), [Eureka](https://github.com/xmartlabs/Eureka), [Moya](https://github.com/Moya/Moya), [RxSwift](https://github.com/ReactiveX/RxSwift), [SnapKit](https://github.com/SnapKit/SnapKit)) it seems like there has been wide adoption of Swift 5. I am comfortable with merging this now.

Thank you for putting in the work for this @ikait 🙌

**Review by sherwinski — APPROVED** [review]

**Pull request merged**